src/main/java/de/dhbw/mh/slang/antlr/AstBuilder.java

```diff
@@ -60,6 +65,15 @@ public AstNode visitExponentiation( ExponentiationContext ctx ){
 	
 	@Override
 	public AstNode visitAtomicExpression( AtomicExpressionContext ctx ){
+		if(ctx.IDENTIFIER() != null){
+			return new AstVariable(ctx.getText());
+		}
+		if(ctx.NUMERIC_LITERAL() != null){
+			return new AstLiteral(NumericalEvaluator.parse(ctx.getText()));
+		}
+		if(ctx.LPAREN() != null){
+			return this.visitLogicalOrExpression(ctx.logicalOrExpression());
+		}
 		return super.visitAtomicExpression( ctx );
```

Comment:
In my opinion, an exception should be thrown here so that when the grammar is expanded, it is immediately apparent that something needs to be done here. Otherwise, any errors may not be noticed until later.